src/nimble_mpi.cc

```diff
@@ -149,6 +143,15 @@ int parseCommandLine(int argc, char **argv, nimble::Parser &parser)
       parser.SetToUseTpetra();
 #else
       std::cerr << "\n Flag '--use_tpetra' ignored \n\n";
+#endif
+      continue;
+    }
+    //
+    if (my_arg == "--use_uq") {
```

Comment:
there are specialized commands in the input that invoke the use of uq .... but now that I think about it the command line switch will be useful to run the same input w/o the UQ sampling